handlers/tokeninfo/jwt/tokeninfo.go

```diff
@@ -166,11 +265,11 @@ func getClaim(t *jwt.Token, claim string) (interface{}, bool) {
 	if claims, ok := t.Claims.(jwt.MapClaims); ok {
 		c, has := claims[claim]
 		if !has {
-			log.Printf("Missing claim %q for token %v", claim, t.Raw)
+			log.Printf("Missing claim %q for token", claim)
```

Comment:
@vroldanbet sure, but does it mean secure?

But do the math, what's your alphabet and how long are your words?
For JWT token I guess it's a pretty huge set of valid JWT tokens that an attacker has to precompute.